autoload/wiki/tags.vim

```diff
@@ -535,3 +535,36 @@ function! g:wiki#tags#default_parser.make(taglist, ...) dict abort
 endfunction
 
 " }}}1
+
+
+" {{{1 let g:wiki#tags#orgmode_parser = ...
+let g:wiki#tags#orgmode_parser = {
```

Comment:
Would it make sense to reuse code from the default parser, i.e. like this:

```vim
let g:wiki#tags#orgmode_parser = deepcopy(g:wiki#tags#default_parser)
let g:wiki#tags#orgmode_parser.X = Y
```

Reply:
I've given it a try and the length and number of regexes make it a bit difficult to fit this pattern within 79 columns. Let me know if the below looks good to you and I'll commit it:

```vim
" {{{1 let g:wiki#tags#orgmode_parser = ...
let g:wiki#tags#orgmode_parser = deepcopy(g:wiki#tags#default_parser)
call extend(g:wiki#tags#orgmode_parser, {
      \ 're_match': '\v^%(\c#\+filetags:\s+:[^: ]+:|\*.*:[^: ]+:$)',
      \ 're_findstart': '\v^%(^\c#\+filetags:\s+|\*.*)(:\zs[^: ]+)+$',
      \ 're_parse': '\v:\zs[^: ]+\ze:',
      \ 're_prefix': '\v^\zs.{-}\ze:([^: ]+:)+$'
      \})

function! g:wiki#tags#orgmode_parser.make(taglist, tagline) dict abort
  let l:prefix = matchstr(a:tagline, self.re_prefix)
  let l:tags = empty(a:taglist) ? '' : (':' . join(a:taglist, ':') . ':')
  return trim(l:prefix . l:tags)
endfunction

" }}}1
```

Reply:
Yes, that looks very good to me!